cms-contentful/README.md

```diff
@@ -0,0 +1,22 @@
+Sunrise Java Contentful CMS
+===========================
+
+Module for [Sunrise Java](https://github.com/sphereio/commercetools-sunrise-java) with Contentful service. 
+
+### How to use it
+
+To implement Contentful data into sunrise project, 
+one need to create ContentfulCmsService object using API token and space id:
+
+`ContentfulCmsService cmsService = ContentfulCmsService.of("spaceId", "token");`
+
+It will automatically fetch all entries.
```

Comment:
Hmm, could be a second class to use contentful, I would have expected that for the cms service get request an API call is fired to contentful. Fetching all data from the CMS could be too much for memory. Also I think commercetools want to be able to change sth. in contentful and then see the changed result in the online shop.